### ISSUE TITLE
Change Command Syntax

### ISSUE THREAD

**Issue by Enthalpy-AC** [Enthalpy-AC/catalysis#1]
Currently, commands occupy a string on a single line. For longer commands, especially ones employing expression mode, this can easily become unwieldy. Here's a simple use case of display two pieces of evidence:

dispEv, foo, c, $'profiles', 1+shift_if_profile_changed, lr

It's hard to tell where one piece of evidence ends and the other begins, and which sections exist through advanced mode. The syntax should resolve these issues.

Proposed ideas include putting different "pieces" on different lines, or some mandatory grouping operator. No answer has been settled on, and there should be agreement among users on syntax before development begins.

---

The following implementation has been accepted. Please refer to the expression_pack terminology of library_aux.py for definitions of the terms unit and subunit:

**RULES:**
A single action (command with the @action decorator) was formerly represented as one line in frame_data.txt, but will now be on multiple.
The first line of the action is a series of arguments separated by a comma and a space. The first value is the function name, and the remaining values are the global arguments (the global unit).
Each multiple value goes on the next line. This proceeds until a terminal colon is reached. This colon can be on the global line, if there are no multiples.
To activate advanced mode for a subunit, the initial $ is replaced with curly braces {} around the subunit. Both braces must be escaped when being used literally.
All actions MUST be the last command called on that frame. To do otherwise would force some "end action" part of the syntax.
Enclosing a word in $ remains the way to refer to a Catalysis variable.

**SUMMARY:**
Each unit breaks onto its own line, and each expression-subunit is now in braces. Until you end a line with a colon, Catalysis will keep parsing your lines as new multiple units. Any uses of a literal brace must be escaped.

**EXAMPLES:**

```
proceed, foo:
->
proceed, foo:

proceed, foo
color, green:
->
color, green
proceed, foo:

choice, BraceString{, my_anc:
->
choice, BraceString\{, my_anc:

dispEv, foo, c, $'profiles', 1+shift+$bar$, lr:
->
dispEv
foo, c
{'profiles', 1+shift+$bar$}, lr:

dispEv, foo, c, $'profiles', 1+shift+$bar$, $lr:
->
dispEv
foo, c
{'profiles', 1+shift+$bar$}, {lr}:
```

**Comment by Enthalpy-AC** [comment]
Added a target implementation.

**Comment by Enthalpy-AC** [comment]
This issue is now blocking Issue #7, a lower priority syntax change for expression mode, a special case of command syntax. That issue may be disregarded for solving this issue, but trying to change the general command syntax simultaneously with expanding a special case of command syntax will not end well.

**Comment by Enthalpy-AC** [comment]
This issue will (probably) be resolved in two pieces - a half-fix to canonize the use of curly braces, and a much deeper half-fix to put "multiple" units on different lines.

**Comment by Enthalpy-AC** [comment]
Put the examples in a code block. In addition to more clearly marking that section, the escape-\ is no longer hidden.

**Comment by Enthalpy-AC** [comment]
Fixed in commit b1f1644.
